pkgs/tools/misc/diffoscope/default.nix

```diff
@@ -169,55 +168,50 @@ python3.pkgs.buildPythonApplication rec {
     pyxattr
     rpm
     tlsh
+  ]) ++ lib.optionals enableBloat ([
+    abootimg
+    acl
+    apksigcopier
+    apksigner
+    cbfstool
+    colord
+    enjarify
+    ffmpeg
+    fpc
+    ghc
+    ghostscriptX
+    giflib
+    gnumeric
+    gnupg
+    hdf5
+    imagemagick
+    jdk
+    libcaca
+    llvm
+    mono
+    ocaml
+    odt2txt
+    oggvideotools
+    openssh
+    pdftk
+    poppler_utils
+    procyon
+    qemu
+    R
+    radare2
+    tcpdump
+    ubootTools
+    wabt
+    xmlbeans
+  ] ++ (with python3.pkgs; [
+    androguard
+    binwalk
+    guestfs
+    h5py
+    pdfminer-six
   ])
-  ++ lib.optionals enableBloat (
-    [
-      abootimg
-      apksigcopier
-      apksigner
-      cbfstool
-      colord
-      enjarify
-      ffmpeg
-      fpc
-      ghc
-      ghostscriptX
-      giflib
-      gnupg
-      hdf5
-      imagemagick
-      jdk
-      libcaca
-      llvm
-      mono
-      ocaml
-      odt2txt
-      openssh
-      pdftk
-      poppler_utils
-      procyon
-      qemu
-      R
-      radare2
-      tcpdump
-      ubootTools
-      wabt
-      xmlbeans
-    ]
-    ++ (with python3.pkgs; [
-      androguard
-      binwalk
-      guestfs
-      h5py
-      pdfminer-six
-    ])
-    # oggvideotools is broken on Darwin, please put it back when it will be fixed?
-    ++ lib.optionals stdenv.isLinux [ oggvideotools ]
```

Comment:
What's the deal with removing all of those? Have you tested them?

Reply:
I got tripped by the comment at the beginning of pythonPath which only filters by platform, not by broken.

Reply:
I got the same thought a while ago, so I added a comment so that this does not happen a third time.